ui/src/app/cohort-search/tree/tree.component.tsx

```diff
@@ -193,7 +193,10 @@ export const CriteriaTree = fp.flow(withCurrentWorkspace(), withCurrentConcept()
       const {cdrVersionId} = currentWorkspaceStore.getValue();
       const criteriaType = domainId === Domain.DRUG.toString() ? CriteriaType.ATC.toString() : type;
       const parentId = domainId === Domain.PHYSICALMEASUREMENT.toString() ? null : id;
-      const promises = [cohortBuilderApi().findCriteriaBy(+cdrVersionId, domainId, criteriaType, isStandard, parentId)];
+
+      const promises = (domainId === Domain.VISIT.toString())
+          ? [cohortBuilderApi().findCriteriaBy(+cdrVersionId, domainId, criteriaType, isStandard)]
```

Comment:
The previous line (195) is basically doing the same thing for Physical Measurements, so we can remove that one and consolidate both here. Also, according to Brian, we don't need `isStandard` for either of those. I tested and seems to work:
```suggestion
      const promises = (domainId === Domain.PHYSICALMEASUREMENT.toString() || domainId === Domain.VISIT.toString())
          ? [cohortBuilderApi().findCriteriaBy(+cdrVersionId, domainId, criteriaType)]
```

Reply:
Done